opsdroid/configuration/example_configuration.yaml

```diff
@@ -14,27 +14,27 @@
 ## A default config file to use with opsdroid
 
 ## Set the logging level
-# logging:
-#   level: info
-#   path: opsdroid.log
-#   console: true
+#logging:
+#  level: debug
```

Comment:
Can you please revert the styling changes here and set this back to info? Looks like a bunch of spaces have been removed.